spinn_machine/core_subset.py

```diff
@@ -105,6 +105,6 @@ def intersect(self, other):
         """
         result = CoreSubset(self._x, self._y, [])
         for processor_id in self._processor_ids:
-            if processor_id in other._processor_ids:
+            if processor_id in other.processor_ids:
```

Comment:
I don't like this change, as it turns a hash lookup (fast) into a scan through an iterable. Fortunately the underlying collection will never be all that large, but I still object to the principle of such a change.

This is a case where suppressing the pylint warning is the right approach.

Reply:
I agree! Changed